index.html

```diff
@@ -414,11 +414,162 @@ <h6> Category: Men's t-shirt</h6>
+               sandbox="allow-scripts allow-same-origin">
+            </iframe>
+            
+
+        </div>
+
+
+          <!-- --------------------------------------------------- -->
+
+
+          <h2 class = "responsiveheading">Input with validation & Error style</h2>
+            <div class="inputContainer3">
+                    
+                
+                <div>
+                    <div class="input-container">
+                        <label class = "label-position" for="email-id">Email address</label>
+                        <input class="user-input" type="email" name="" id="email-id" placeholder="enter your email" value = "rahul@neogcamp.com">
+                    
+                        <span id = "check"><i class="fas fa-check"></i></span>
+                    
+                        </div>
+                        <div id = "password-outer">
+                          <div id = "password-inner">
+                        <label  class = "label-position" for="user-password">Password</label>
+                            <input class="user-input" type="password" name="" id="user-password" placeholder="enter your password" value = "1234">
+                            <span class = "wrong"><i class="fas fa-times"></i></span>
+                            </div>
+                          <small class = "wrong">Password must have atleast 1 alphaneumeric & 1 special character!</small>
+                        </div>  
+
+                </div>
+
+            </div>
+          </div>  
+         
+          
+    </div>  
```

Comment:
Use a code formatter to remove these gaps in code and make it cleaner.

Reply:
Yes in real world projects ,it is mandatory to avoid extra spaces.